api/src/main/java/org/openmrs/api/ValidationException.java

```diff
@@ -91,11 +99,4 @@ public ValidationException(String message, Errors errors) {
 	public Errors getErrors() {
 		return errors;
 	}
-	
-	/**
-	 * @since 1.11
-	 */
-	public void setErrors(Errors errors) {
-		this.errors = errors;
-	}
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation ;)

Reply:
Thank you, will fix this immediately.

Reply:
@sacull I am kind of confused now,  wasn't I supposed to remove that one?